l/linux-pam/stone.yaml

```diff
@@ -66,3 +66,6 @@ install     : |
 
     %tmpfiles "# Create run/faillock directory"
     %tmpfiles "d /run/faillock 0755 root root -"
+
+    chmod u+s %(installroot)/usr/sbin/unix_chkpwd
```

Comment:
Please add a comment above so people don't need to git blame to understand why it is necessary to use setuid here.